tests/testdata/py-no-tls-no-run-line-no-code.in

```diff
@@ -0,0 +1 @@
+--no-tlc --no-code --no-runline file.py
```

Comment:
The name of this file should be `py-no-tlc` (not tls) `-no-runline-no-code.in` (i.e., mimic the flags on the command line). Similarly for the *.out and *.err files, of course.